### ISSUE TITLE
insert entity panel should open with keyboard focus in "search entities" text field

### ISSUE THREAD

**Issue by movesmyers** [textmate/html.tmbundle#25]
when opening the "insert entity" panel, keyboard focus should be in the "search entities" text field enabling the user to search for an entity without first clicking in the search box.
